core/main/src/broker/endpoint_broker.rs

```diff
@@ -641,12 +644,11 @@ impl BrokerOutputForwarder {
                                             }
                                             response.id = Some(request_id);
 
-                                            let message = ApiMessage {
-                                                request_id: request_id.to_string(),
+                                            let message = ApiMessage::new(
                                                 protocol,
-                                                jsonrpc_msg: serde_json::to_string(&response)
-                                                    .unwrap(),
-                                            };
+                                                serde_json::to_string(&response).unwrap(),
```

Comment:
unsafe unwrap here. Use pattern matching instead
`let response = match serde_json::to_string(&response) {...}`

Reply:
Existing code and its Serialization of an existing rust object. Even the underlying Serde impl only returns OK

```
#[inline]
pub fn to_string<T>(value: &T) -> Result<String>
where
    T: ?Sized + Serialize,
{
    let vec = tri!(to_vec(value));
    let string = unsafe {
        // We do not emit invalid UTF-8.
        String::from_utf8_unchecked(vec)
    };
    Ok(string)
}
```

Reply:
Still it's a good practice to use the pattern matching code style. We have cleaned up many `unwrap()` usage with pattern matching.